crypto/crypto.go

```diff
@@ -25,7 +25,10 @@ func GeneratePrivateKey() (PrivateKey, error) {
 }
 
 func GenerateDifficultKey(difficulty uint) (PrivateKey, error) {
-	for true {
+	if difficulty > 256 {
+		return nil, errors.New("Difficulty to high")
```

Comment:
`s/to/too`
Also the convention for error messages in Go is to start them with non capital letter in cases like this (so that later you can do `fmt.Println("An error occurred:", err)` and you won't get capital letter in the middle of the sentence